### PR TITLE
EN 979 delete groups confirmation

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

File: sass/groups/common/DeleteGroupModal.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.DeleteGroupModal {
+  &__info {
+    font-size: 1.7em;
+  }
+}
+
+.DeleteGroupModal__icon {
+  span {
```
Comment (alessioscalici) [inline sass/groups/common/DeleteGroupModal.scss]: Please do not use tag elements as selectors. This is highly inefficient (the browser selects all spans, then filters out spans that are not descendant of .DeleteGroupModal__icon).
There's also a linter rule against this, I wonder why it did not work this time :(

File: src/api/groups.js [hunk 2]
```diff
@@ -51,4 +51,13 @@ export const postGroup = groupObject => (
   })
 );
 
+export const deleteGroup = groupCode => (
+  makeMockRequest({
+    uri: `/api/groups/${groupCode}`,
+    method: METHODS.DELETE,
+    mockResponse: { payload: { code: 'groupCode' } },
```
Comment (alessioscalici) [inline src/api/groups.js]: Payload is already included by makeMockRequest, please remove:

`{ payload: { code: 'groupCode' } }` -> `{ code: 'groupCode' }`

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alessioscalici** [comment]
retest this please

**Pull request merged**